magento/pwa-studio
# [Documentation] Toasts reference docs

## Description

Create documentation for the toasts hooks and venia component

## Related Issue

Closes #1342 

## Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Build the HTML: `npm run build`
3. Verify the following docs are autogenerated:
    * `src/_includes/auto-generated/venia-ui/lib/components/ToastContainer/toastContainer.md`
    * `src/_includes/auto-generated/peregrine/lib/Toasts/useToasts.md`
    * `src/_includes/auto-generated/peregrine/lib/Toasts/useToastContext.md`
4. Verify the auto-generated docs have content based on the doc blocks in the source files.

**Note:** There is no need to preview server since this content will not be published until the component is published to NPM.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
